### PR TITLE
refactor(dh): migrate to new built-in control flow syntax

### PR DESCRIPTION
**Pull request by dzhavat**
## References

- N/A

### CODE REVIEW

File: libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-audit-logs.component.html [hunk 1]
```diff
@@ -15,12 +15,14 @@
...
-  </div>
+  @if (isLoading$ | push) {
+    <div class="spinner">
+      <watt-spinner />
+    </div>
+  }
 
-  <ng-container *ngIf="!(isLoading$ | push)">
-    <ng-container *ngIf="!(hasFailed$ | push)">
+  @if (!(isLoading$ | push)) {
```
Comment (mimse) [inline libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-audit-logs.component.html:24]: Suggestion: Combine these two in to one if?

File: libs/dh/admin/feature-user-management/src/lib/drawer/tabs/content/dh-user-audit-logs.component.html [hunk 2]
```diff
@@ -15,12 +15,14 @@
...
-  </div>
+  @if (isLoading$ | push) {
+    <div class="spinner">
+      <watt-spinner />
+    </div>
+  }
 
-  <ng-container *ngIf="!(isLoading$ | push)">
-    <ng-container *ngIf="!(hasFailed$ | push)">
+  @if (!(isLoading$ | push)) {
```
Comment (mimse) [inline libs/dh/admin/feature-user-management/src/lib/drawer/tabs/content/dh-user-audit-logs.component.html:24]: Suggestion: Combine in to one if?

**Review by mimse — CHANGES_REQUESTED** [review]

File: libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-audit-logs.component.html (hunk 1 above)
Comment (dzhavat) [reply libs/dh/admin/feature-permissions/src/lib/details/tabs/dh-admin-permission-audit-logs.component.html]: Should be fixed now.

File: libs/dh/admin/feature-user-management/src/lib/drawer/tabs/content/dh-user-audit-logs.component.html (hunk 2 above)
Comment (dzhavat) [reply libs/dh/admin/feature-user-management/src/lib/drawer/tabs/content/dh-user-audit-logs.component.html]: Should be fixed now.

**Review by mimse — APPROVED** [review]

**Pull request merged**